content/blog/cosign-2.5.0.md

```diff
@@ -0,0 +1,132 @@
+   In our setup we use [offline installer](https://goharbor.io/docs/2.5.0/install-config/download-installer/).
+
+```
+# ./install.sh --with-notary --with-trivy
+
+[Step 0]: checking if docker is installed ...
+...
+...
+...
+[Step 5]: starting Harbor ...
+Creating network "harbor_harbor" with the default driver
+Creating network "harbor_harbor-notary" with the default driver
+Creating network "harbor_notary-sig" with the default driver
+Creating harbor-log ... done
+Creating harbor-portal ... done
+Creating redis         ... done
+Creating registryctl   ... done
+Creating registry      ... done
+Creating harbor-db     ... done
+Creating trivy-adapter ... done
+Creating notary-signer     ... done
+Creating harbor-core   ... done
+Creating harbor-jobservice ... done
+Creating nginx             ... done
+Creating notary-server     ... done
+✔ ----Harbor has been installed and started successfully.----
+```
+
+#### 2. Create new project "cosign"
+![Cosign Project](../img/cosign/cosign-project.png)
+Enable cosign on the repo:
+![Cosign cosign](../img/cosign/cosign-cosign.png)
+
+#### 3. Create new user "cosign-demo" and assign it to the project "cosign"
+![cosign-demo](../img/cosign/cosign-user.png)
+![cosign-demo-member](../img/cosign/cosign-member.png)
+
+#### 4. Setup project cosign on second instance
+![Cosign Project](../img/cosign/cosign-project.png)
+
+#### 5. [Create robot account](https://goharbor.io/docs/2.5.0/administration/robot-accounts/) on second instance "robo-cosign"
+![robo-cosign](../img/cosign/cosign-robot.png)
+Save robot secret for setup replication:
+![robo-secret](../img/cosign/cosign-robo-secret.png)
+#### 6. [Setup replication](https://goharbor.io/docs/2.5.0/administration/configuring-replication/) on the first instance
+Setup new target registry:
+![harbor2](../img/cosign/cosign-harbor2.png)
+
+Setup Replication from harbor1 -> harbor2:
+![replication](../img/cosign/cosign-replication.png)
+
+#### 7. Create cosign key-pair
+```
+$ cosign generate-key-pair
+Enter password for private key:
+Enter again:
+Private key written to cosign.key
+Public key written to cosign.pub
+```
+
+You can export the password for the key(aka pass-phrase) so you can use it in automation:
+```
+export COSIGN_PASSWORD=Your_Super_P1$$w0rD
+```
+
+#### 8. Push and sign Image
+```
+$ docker login harbor1.orlix.org
+Authenticating with existing credentials...
+Login Succeeded
+
+$ docker push harbor1.orlix.org/cosign/pause:1
+The push refers to repository [harbor1.orlix.org/cosign/pause]
+5f70bf18a086: Layer already exists
+e16a89738269: Layer already exists
+1: digest: sha256:b31bfb4d0213f254d361e0079deaaebefa4f82ba7aa76ef82e90b4935ad5b105 size: 938
+
+$ cosign sign --key cosign.key harbor1.orlix.org/cosign/pause:1
+Enter password for private key:
+Pushing signature to: harbor1.orlix.org/cosign/pause
+```
+
+#### 9. Trigger replication and verify results
+![harbor1-replicate](../img/cosign/cosign-replicate.png)
+
+Verify replication:
```

Comment:
it could also add the `cosign --verify` section to in the harbor2?

Reply:
added!